picard/util/__init__.py

```diff
@@ -39,7 +39,6 @@
     IS_WIN,
 )
 
-
```

Comment:
The number of whilelines is rather confusing, that's 2, but only for functions or classes. So not in this case.

Reply:
Not sure what you mean. Are you suggesting to always use two lines, even for cases like this? Can this be configured with isor5?

Reply:
Not sure if desirable, and whether it's compatible with PEP8 auto fixers. But
https://github.com/timothycrosley/isort/wiki/isort-Settings

> lines_after_imports: Forces a certain number of lines after the imports and before the first line of functional code. By default this is 2 lines if the first line of code is a class or function. Otherwise it's 1.

Reply:
I added this in  dc0ec84401150d54a121ff69bd6d21de035d2e54, I think it makes sense. At least flake8 is happy with both variations.